<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>240</horstretch>
    <verstretch>68</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>680</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Remote Control</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>png/speakers.png</normaloff>png/speakers.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>480</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutLCD">
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdVol">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="intValue" stdset="0">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelVolume">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Volume</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="lcdChannel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Channel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Channel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>231</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout0_9">
     <item row="0" column="2">
      <widget class="QPushButton" name="button3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QPushButton" name="buttonVolMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icon.qrc">
         <normaloff>:/png/icon-minus6.png</normaloff>:/png/icon-minus6.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="buttonChannelUp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icon.qrc">
         <normaloff>:/png/pointing-up.png</normaloff>:/png/pointing-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="button4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="button9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QPushButton" name="buttonVolAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icon.qrc">
         <normaloff>:/png/icon-add.png</normaloff>:/png/icon-add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="buttonVolOff">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icon.qrc">
         <normaloff>:/png/volume-up.png</normaloff>
         <normalon>:/png/volume-down.png</normalon>:/png/volume-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="button7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="button0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="button6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="button5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="button8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QPushButton" name="buttonChannelDown">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Icon.qrc">
         <normaloff>:/png/pointing-down.png</normaloff>:/png/pointing-down.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonVolMinus</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>volMinus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonVolAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>volAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonVolOff</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>volOff()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonChannelDown</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>channelMinus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonChannelUp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>channelAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>-8</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>-6</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button9</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>volAdd()</slot>
  <slot>volMinus()</slot>
  <slot>volOff()</slot>
  <slot>channelAdd()</slot>
  <slot>channelMinus()</slot>
  <slot>button1()</slot>
  <slot>button2()</slot>
  <slot>button3()</slot>
  <slot>button4()</slot>
  <slot>button5()</slot>
  <slot>button6()</slot>
  <slot>button7()</slot>
  <slot>button8()</slot>
  <slot>button9()</slot>
  <slot>button0()</slot>
 </slots>
</ui>
